adcc/adc_pp/modified_transition_moments.py

```diff
@@ -30,47 +30,54 @@
 from adcc.AmplitudeVector import AmplitudeVector
 
 
-def mtm_adc0(mp, dipop, intermediates):
-    return AmplitudeVector(ph=dipop.ov)
+def mtm_adc0(mp, op, intermediates):
+    return AmplitudeVector(ph=op.ov) if op.is_symmetric \
+        else AmplitudeVector(ph=op.vo.transpose())
 
 
-def mtm_adc1(mp, dipop, intermediates):
-    f1 = dipop.ov - einsum("ijab,jb->ia", mp.t2(b.oovv), dipop.ov)
+def mtm_adc1(mp, op, intermediates):
+    f1 = op.ov if op.is_symmetric else op.vo.transpose()  # zeroth order
```

Comment:
Better not repeat. Just call `mtm_adc0` to reuse the expression from above. Also makes the structure of the equations and the orders more explicit. If I remember correctly we do this for the matrix equations as well. (have not checked)